src/masternodes/govvariables/lp_daily_dfi_reward.cpp

```diff
@@ -18,8 +18,12 @@ UniValue LP_DAILY_DFI_REWARD::Export() const {
     return ValueFromAmount(dailyReward);
 }
 
-Res LP_DAILY_DFI_REWARD::Validate(const CCustomCSView &) const
+Res LP_DAILY_DFI_REWARD::Validate(const CCustomCSView & view) const
 {
+    if (view.GetLastHeight() >= static_cast<uint32_t>(Params().GetConsensus().EunosHeight)) {
```

Comment:
We set last height just before `ConnectBlock` ends, so in Eunos height `GetLastHeight` will point to previous one, do you think? It can be used by -1 at end.

Reply:
An off by one in this circumstance does not make a difference, we do not expect to set LP_DAILY_DFI_REWARD manually and especially not on the hard fork block. The automatic changing of the LP_DAILY_DFI_REWARD comes after parsing of the setgov custom TX.